recipes/LibriSpeech/ASR/CTC/hparams/train_hf_wav2vec_k2.yaml

```diff
@@ -89,17 +90,21 @@ sample_rate: 16000
 # With DDP batch_size is multiplied by N jobs
 # Must be 3 per GPU to fit 32GB of VRAM
 batch_size: 6
-test_batch_size: 1
+test_batch_size: 6
```

Comment:
Regarding `test_batch_size=6`, do you think that it is suitable for GPUs with small memory?

Reply:
Hello, you should not change the batch size from 1. The reason is not about gpu consumption but about results. Indeed, having a batch_size > 1 will lead to tails effect on the convolutions and might harm the final wer.

Reply:
> Regarding test_batch_size=6, do you think that it is suitable for GPUs with small memory?

If you can train with `batch_size: 6`, you should be able to decode with the same batch_size.
If you only want to decode, with `test_batch_size=6` the gpu consumption is less than 9Gb for the 6 largest audio segments (sorting=ascending). I run this for HL onebest decoding. I'll test later for HLG whole-latice-rescoring.
(I added `torch.cuda.empty_cache()` after each forward, as by default, the pytroch GPU GC is not triggered if there is enough memory on the GPU, should I also commit this to the PR ?)

 > Hello, you should not change the batch size from 1. The reason is not about gpu consumption but about results. Indeed, having a batch_size > 1 will lead to tails effect on the convolutions and might harm the final wer.

I don't know how it works for the others egs of sb, but with k2 it works well.
I did not observe any significant wer degradation.
Do you want me to set `test_batch_size` to one?

Reply:
Yes please change it to 1. I already did some experiments and it can harm the wer significatively when increasing the batch size :/